### ISSUE TITLE
Question: Chromosome-wide zscore to use?

### ISSUE THREAD

**Issue by biocyberman** [rstraver/wisecondor#22]
I am new to wisecondor. Reading the publication, analysis .txt output file and the source code itself, I am not sure where to get overall chromosome Z-Score. Looking at the excerpt of ouput below for example, the "Chromosome wide, aneuploidy test" is actually Stouffer' z-score. How do I get a normal z-score for the whole chrosomome?

> Windowed, bin test:
> 	Change	Multi	Z-Score	Size	Position
> 	+	1.064	7.135	34Mb	chr21:14000000-48000000
> Found:	1	Filtered:	1
> 
> Windowed, aneuploidy test:
> 	chr21: 100.0% marked
> 
> Chromosome wide, aneuploidy test:
> 	Chr14	-3.04959010798	0.991971731433
> 	Chr21	14.6839341651	1.06444989956

**Comment by rstraver** [comment]
Hi!

Have you found the wiki page on this topic yet?: https://github.com/rstraver/wisecondor/wiki/Understanding-the-output

Basically, the _Chromosome wide, aneuploidy test_ provides a Stouffers z-score for the whole chromosome but that is likely way too sensitive for small variations. We just included it as a backup to look into chromosomes more closely when there is a fairly large z-score there but no mention of aberrations in the _Windowed, bin test_. If you are to find trisomies and such, I'd suggest thresholding the percentage marked by the windowed test as provided by _Windowed, aneuploidy test_.

If what you ask for is just a more straightforward single value for the whole chromosome, _Chromosome wide, aneuploidy test_ is the closest thing you can get due to how WISECONDOR is implemented, but again, a standard threshold of 3 might be too sensitive. If you look at it's output, you'll see it provides two numbers per chromosome call:
> Chr14 -3.04959010798 0.991971731433
> Chr21 14.6839341651 1.06444989956 

Basically, it says `chromosome | z-score | mean-deviation`. The last value thus shows chr14 was found deviating with a loss of: `(0.991971731433-1)*100=-0.8028268567`
So -0.8%. That's an extremely small number and likely caused by experimental effects.
However, chromosome 21 is increased by 6.4%, which is reasonable to assume being caused by a trisomy. Additionally, it received a much higher z-score so it's much more likely a true call than the other chromosome.

As trying to tell trisomy from other effects per sample is difficult from just a few numbers for a whole chromosome we generally look at the output plots instead of just the numbers and see if any stretches of z-scores are found steadily increased or decreased, which are marked by purple on the top of a plot.

**Comment by biocyberman** [comment]
No, I missed the wikipage. Thanks for the explanation.
